scripts/basic-workflow-tests.sh

```diff
@@ -110,8 +110,14 @@ test_smoke() {
 
 test_init() {
     task Initializing ilab
-    ilab config init --non-interactive
 
+    if [ "$FULLTRAIN" -eq 1 ]; then
```

Comment:
We should prefer to use `[[` and `]]` instead of `[` and `]` since it reduces errors. See the guideline on [`[[` vs `[`](https://google.github.io/styleguide/shellguide.html#test----and---). 

```suggestion
    if [[ "${FULLTRAIN}" -eq 1 ]]; then
```